Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS build
+
+WORKDIR /app
+
+COPY . /app
+
+FROM python:${PYTHON_VERSION} AS run
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=build /app .
```

Comment:
The COPY --from=build command should specify the correct source and destination paths. Ensure that the paths are correct and that the necessary files are copied from the build stage.